Safewaan/HonkedIn
# Styling for the Navigation Bar

## Trello ticket link

[Developer Task: Implement UI/UX feedback to the navigation bar](https://trello.com/c/5IA61xjQ/119-developer-task-implement-ui-ux-feedback-to-the-navigation-bar )



## Implementation description
* Added new component framework, Chakra UI
* Rebuilt the Navigation Bar



## Steps to test
Routing Tests
1. Run `npm i` to install new deps
2. Verify that all menu options open their corresponding menu lists
3. Verify that each menu list option routes to the correct page
4. Verify there are no visual conflicts with other pages

Style Tests
1. Verify that hovering over menu items changes the background colour to #164684
2. Verify that clicking on a menu item changes the background colour to #214E89
3. Verify that hovering over a menu list item changes the font colour to #023679
4. Optional: ensure styling matches up with the [Figma](https://www.figma.com/file/9HWk5TlYt0JyuBmSKRskeC/Safwaan-Schoel-Preojnfnct?node-id=1-4&t=ni9RP11P2Jp9ZeCU-0 ) mock

Visual Tests
1. Verify all margins are correct, including margins between menu items and icons (such as chevrons or add icons)
2. Verify that all pages that display Alerts function correctly



## What should reviewers focus on?
* Are there colour schemes we should adopt? Fonts and font sizes?
